### PR TITLE
[RFC] Alternative Animal Version

### PR DESCRIPTION
**Pull request by gnunicorn**
While I love the idea of using emoji to highlight more of the work done by contributors – especially outside of the projects main source, I've found the selected icons do emphasize too strongly on the work rather than the person behind it and appear quite serious on the way they do it. Like "badges" to be earned.

This is an alternative way of looking at the idea. Instead of using icons to represent the work, this edition exclusively uses animals, animal faces and funny names to describe the work done by the person – in form of a cute animal they are represented by: 🐵 🐶 🐲 🕊 🐴 🐙 🐰 🐬 🦄 🐼 🐝

This is not meant to be merged as is, but rather a discussion document. I really just threw this together in a few minutes to give an idea if it was feasible and what it would look like. This current version is neither complete nor without judgement (see "code monkey") and should be taken with a grain of salt.

You can find a rendered version at https://github.com/ligthyear/all-contributors/blob/alt-animals/Spec-Animals.md

What do you think about this?

### CODE REVIEW

**Comment by Jameskmonger** [comment]
Looks great to me. I love it!

File: Spec-Animals.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# ✨ All Contributors Ainmals v1.0.0-beta.0 ✨
+
+The general specification as defined in the [README](./README.md) applies, but in this alternative version, we replaced all emoji icons with cue animals.
```
Comment (ben-eb) [inline Spec-Animals.md]: *cute animals.*

File: Spec-Animals.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+🐲 | Translator Dragon | Any translation work done | the translated content
+🕊 | Designer Dove | Any form of design work | the logo/iconography/visual design/etc.
+🐴 | Helpful Horsie | Any Infrastructure Work, anything on deploy, automatising or build pipeline | link to the code, if applicable
+🐙 | Octestopos | Provides Tests | `https://github.com/${ownerName}/${repoName}/commits?author=${username}`
+🐰 | Thorough Rabbit | Bug reporting and Feedback | `https://github.com/${ownerName}/${repoName}/issues?q=author%3A${username}`
+🐬 | Helpful Dolphin | Answering Questions and helping people (in Issues, Stack Overflow, Gitter, Slack, etc.) | link to external profile if applicable
+🦄 | Community Unicorn | Community work like organising events, meetups and team communication | link to events or profiles to contact, if applicable
+🐼 | Outspoken Panda | External Communication: blog posts, videos, talks, any form of project reach out | link to material, if applicable
+🐝 | Bussy Bee | Reviewing PRs, organising tickets, project coordination and communication
```
Comment (ben-eb) [inline Spec-Animals.md]: *Busy Bee*

**Comment by kentcdodds** [comment]
I also love it. One concern I'll add to the inline comments.

File: README.md [hunk 3]
```diff
@@ -62,10 +64,10 @@ Emoji | Represents | Links to
...
.../commits?author=bogas04) 👀 | [![Ben Briggs](https://avatars1.githubusercontent.com/u/1282980?v=3&s=100)<br /><sub>Ben Briggs</sub>](http://beneb.info)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb) 👀 | [![James Monger](https://avatars1.githubusercontent.com/u/2037007?v=3&s=100)<br /><sub>James Monger</sub>](http://github.com/Jameskmonger)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger) | [![Jeroen Engels](https://avatars.githubusercontent.com/u/3869412?v=3&s=100)<br /><sub>Jeroen Engels</sub>](https://github.com/jfmengels)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=jfmengels) [🔧](https://www.npmjs.com/package/all-contributors-cli) 👀 | [![Chris Simpkins](https://avatars0.githubusercontent.com/u/4249591?v=3&s=100)<br /><sub>Chris Simpkins</sub>](http://github.com/chrissimpkins)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=chrissimpkins) 👀 | [![Frederic Hemberger](https://avatars0.githubusercontent.com/u/153481?v=3&s=100)<br /><sub>F. Hemberger</sub>](http://github.com/fhemberger)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=fhemberger) |
```
Comment (kentcdodds) [inline README.md]: What are you changing about these lines?

File: README.md [hunk 4]
```diff
@@ -62,10 +64,10 @@ Emoji | Represents | Links to
...
 | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
 
-| [![Daniel Kraft](https://avatars1.githubusercontent.com/u/3982200?v=3&s=100)<br /><sub>Daniel Kraft</sub>](http://github.com/frigginglorious)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=frigginglorious) |
```
Comment (kentcdodds) [inline README.md]: this one also

**Comment by chrissimpkins** [comment]
This isn't an idea that I can get behind as the emoji image default.   There will be a crowd for the 'cute' image look but this is likely to narrow the range of projects willing to use specification specified images in their attributions.  Sorry to be the guy who doesn't vote for unicorns but I would recommend keeping this a very optional part of the spec if it is adopted.  Take this comment with a large grain of salt because I am not an emoji fan at all, panda bears or not.  

Now, get off the lawn ;)

**Comment by kentcdodds** [comment]
Oh totally, definitely an optional part of the spec :+1: it doesn't replace the other emoji.

**Comment by ligthyear** [comment]
@chrissimpkins of course. as @kentcdodds mentioned, this is explicitly kept as "an alternative" version. I agree this might not be appropriate for all projects and having only animals – though cute – could inhibit adoption. However, having this _alternative_ playful version could also increase adoption!

Any comments on the specific animals or names picked?

**Comment by bondz** [comment]
Dove and Unicorn emoji not visible on Chrome Android

**Comment by MatheusRV** [comment]
Although I like the idea very much I am against, in human computer interaction when we define a language of communication with the internaut. We must use the same symbol and term for all cases. All-Contributors is perfect the way it is, the language through emoji-related contribution made, is the great differential. Simple and objective. Now thinking of new ways to contribute to a project such as https://github.com/freeCodeCamp/freeCodeCamp/issues/25335 would be amazing.

Vote to close.

**Comment by Berkmann18** [comment]
I like the idea of using animals but that defeats the point of having an HCI friendly set of icons where each icons represent the relevant category.
It would be even harder to maintain having to use animal icons linked to contribution types which itself is not very HCI friendly (memorisation vs real world association + recognition).

**Pull request closed without merging**